layouts/partials/favicon.html

```diff
@@ -0,0 +1,9 @@
+{{- /*TODO: regenerate all the images with this generator: https://realfavicongenerator.net*/ -}}
```

Comment:
Open TODO: currently the icons have been generated with the page's "home icon" and some settings have been set as I saw them fit. You should regenerate the icons with a bigger source file you want as favicon and set the options (e.g. for android and iOS icons) as you like